calcom/cal.com
# fix: changes for showing error message in case of INTERNAL_SERVER_ERROR in event types

## What does this PR do?



Fixes https://github.com/calcom/cal.com/issues/10205



## Type of change




## How should this be tested?



(These steps are from the issue mentioned above)
- In an Event type, click on the Duplicate option
- Set 9000000000 as the length value
- Click on the Continue button
- See the notification error
- Save it with a valid number, like 15
- From the Event Type configuration, set the duration to 9000000000
- Click on the Save button
- See the notification

## Mandatory Tasks


### Before Change
![image](https://github.com/calcom/cal.com/assets/29038590/84f4ecaf-9fbc-4f3a-8b90-e5c7996815e1)

### After Change
![image](https://github.com/calcom/cal.com/assets/29038590/d63826ce-29d7-4a27-b5ca-31004e2dfd60)
